fe/fe-core/src/main/java/com/starrocks/connector/ColumnTypeConverter.java

```diff
@@ -436,6 +436,7 @@ public static Type fromIcebergType(org.apache.iceberg.types.Type icebergType) {
             case TIME:
             case FIXED:
             case BINARY:
+                return Type.VARBINARY;
```

Comment:
convert TIME, FIXED,BINARY to VARBINARY?

Reply:
fixed